.env

```diff
@@ -0,0 +1,2 @@
+TENABLEIO_ACCESS_KEY=
```

Comment:
I'm not sure why these changes are still showing in the repo, can you try rebasing this branch?